admin_guide/manage_users.adoc

```diff
@@ -14,19 +14,60 @@ toc::[]
 [[managing-users-overview]]
 == Overview
 
+A user is an entity that interacts with the {product-title} API. These can be a developer for developing applications or a administrator for managing the cluster. Users can be assigned to groups, which set the permissions applied to all the group's members. For example, you can give API access to a group, which give all members of the group API access.
+
 This topic describes the management of
-xref:../architecture/core_concepts/projects_and_users.adoc#architecture-core-concepts-projects-and-users[user] accounts,
-including how new user accounts are created in {product-title} and how they can
-be deleted.
+xref:../architecture/core_concepts/projects_and_users.adoc#architecture-core-concepts-projects-and-users[user]
+accounts, including how new user accounts are created in {product-title} and how
+they can be deleted.
 
 [[managing-users-adding-a-user]]
-== Adding a User
+== Creating a User
+
+The process for creating a user depends on the configured
+xref:../install_config/configuring_authentication.adoc#identity-providers[identity
+provider]. By default, {product-title} uses the `DenyAll` identity provider,
+which denies access for all user names and passwords. 
 
-After new users log in to {product-title}, an account is created for that user
-per the xref:../install_config/configuring_authentication.adoc#identity-providers[identity
-provider] configured on the master. The cluster administrator can
-xref:../admin_guide/manage_rbac.adoc#admin-guide-manage-rbac[manage the access level of
-each user].
+The following process creates a new user, then adds a role to the user:
+
```

Comment:
Passive. Maybe "Use the following step to create a new user and add a role to the user." ??

Reply:
Hmm I think this one is OK. I don't see how it's passive, and the suggestion is similar.